### PR TITLE
update css from Paul Irish

### PR DESCRIPTION
**Pull request by Gerche**
The css for box-sizing in the article https://www.paulirish.com/2012/box-sizing-border-box-ftw/ was updated.

### CODE REVIEW

File: chapters/1-the-box/index.html [hunk 1]
```diff
@@ -210,11 +210,13 @@ <h3 class="magic">Flexible inputs</h3>
...
-    -webkit-box-sizing: border-box; /* Needed for mobile WebKit */
-    -moz-box-sizing: border-box; /* Needed for Firefox */
-    box-sizing: border-box
-}</code></pre>
+                <pre><code>html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
```
Comment (adamschwartz) [inline chapters/1-the-box/index.html]: Please remove this new line

File: chapters/1-the-box/index.html [hunk 2]
```diff
@@ -210,10 +210,11 @@ <h3 class="magic">Flexible inputs</h3>
 
                 <p>If you want <code>height</code> and <code>width</code> to behave in the most intuitive way, <a href="http://www.paulirish.com/2012/box-sizing-border-box-ftw/">listen to Paul Irish</a><sup><a href="#cite-3">[3]</a></sup> and put this at the top of your CSS:</p>
 
-                <pre><code>*, *:before, *:after {
-    -webkit-box-sizing: border-box; /* Needed for mobile WebKit */
-    -moz-box-sizing: border-box; /* Needed for Firefox */
-    box-sizing: border-box
+                <pre><code>html {
+  box-sizing: border-box;
```
Comment (adamschwartz) [inline chapters/1-the-box/index.html]: Sorry, please remove these `;` as well as that is the way the rest of MoCSS is done

File: chapters/1-the-box/index.html [hunk 3]
```diff
@@ -210,10 +210,11 @@ <h3 class="magic">Flexible inputs</h3>
...
+}
+*, *:before, *:after {
+  box-sizing: inherit;
```
Comment (adamschwartz) [inline chapters/1-the-box/index.html]: And this

**Comment by adamschwartz** [comment]
thx

**Comment by Gerche** [comment]
no problem, thx for the book!

**Pull request merged**